src/stan-users-guide/using-stanc.qmd

```diff
@@ -18,51 +18,52 @@ The stanc3 compiler has the following command-line syntax:
 ```
 > stanc (options) <model_file>
 ```
-where `<model_file>` is a path to either a Stan model
-file ending in suffix `.stan` _or_ a Stan functions
-file which ends in `.stanfunctions`.
+
+where `<model_file>` is either a path to a file ending in
+`.stan` or `.stanfunctions` (which automatically sets
+`--standalone-functions`), or `'-'` to read from standard input.
 
 The stanc3 options are:
 
-- `--help` - Displays the complete list of stanc3 options, then exits.
+- `--help` / `-?` - Displays the complete list of stanc3 options, then exits.
 
 - `--version` - Display stanc version number
 
 - `--info` - Print information about the model, such
-as the type information for variables and the list of used distributions.
+  as the type information for variables and the list of used distributions.
```

Comment:
is a single space right here?  The other ones indent a space past the leading hyphen (e.g., line 38)

Reply:
This is two spaces, I think it just looks odd in the diff